Крестики-нолики_1/PlayField.cpp

```diff
@@ -0,0 +1,124 @@
+#include "PlayField.h"
+#include <cassert>
+
+using namespace std;
+
+PlayField::CellPos::CellPos(int i, int j)
+{
+	assert(i >= 0 && j >= 0 && i <= fieldSize - 1 && j <= fieldSize - 1);
+	x = i;
+	y = j;
+}
+
+vector<PlayField::CellPos> PlayField::getEmptyCells() const
+{
+	vector<CellPos> emptyCell;
+	for (int i = 0; i < fieldSize; i++)
+	{
+		for (int j = 0; j < fieldSize; j++)
+		{
+			if (cellsState[CellPos(i,j).GetPos()] == csEmpty)
+			{
+				CellPos cell = CellPos(i,j);
+				emptyCell.push_back(cell);
+			}
+		}
+	}
+	return emptyCell;
+}
+
+PlayField PlayField::makeMove(CellPos cellPos)
+{
+	return operator+(cellPos);

```

Comment:
```suggestion
	return *(this)+cellPos;

```
Хотелось, конечно, так. Не принципиально.